collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java

```diff
@@ -189,40 +189,24 @@ public static Bitmap getBitmapScaledToDisplay(File file, int screenHeight, int s
         BitmapFactory.Options options = new BitmapFactory.Options();
         options.inJustDecodeBounds = true;
         getBitmap(file.getAbsolutePath(), options);
-
+        // Load full size bitmap image
+        options = new BitmapFactory.Options();
```

Comment:
The new creation of `Option` object can be removed and the one created at the starting of the function should be used